src/Cosi-BLS/block.lisp

```diff
@@ -165,6 +165,31 @@ added to the blockchain."
       blk)))
 
 
+
+(defparameter *newtx-p* t) ; using new-transactions.lisp (package
```

Comment:
@guitarvydas Please ensure that we deprecate `*newtx-p*` as soon as conveniently possible, possibly by confirming when @dbmcclain is ready for it to be removed.  And simulation coordinate the replacement of the `emotiq/sim:run` routine with `emotiq/sim:run-new-tx`.  

Such conditional paths in the source code based on a special at the point when we have no need of "old transactions" are a legendary basis for runtime errors.